packages/react-environment/src/webb-context/mixer/mixer-deposit.ts

```diff
@@ -12,7 +12,7 @@ export type MixerDepositEvents = {
 };
 export type DespotStates = 'ideal' | 'generating-note' | 'depositing';
 
-export type MixerSize = {
+export type MixerTitle = {
```

Comment:
`MixerSize` isn't just for display and more data will be added to it

Reply:
Changed it back, but curious what we will be adding to this field.

Reply:
Don't know in the meantime as the type has the id which can be used to find more related information